frost-ed25519/tests/frost.rs

```diff
@@ -1,20 +1,50 @@
-use curve25519_dalek::{edwards::EdwardsPoint, traits::Identity};
 use frost_core::{Ciphersuite, Group, GroupError};
 use frost_ed25519::*;
+
+use curve25519_dalek::{edwards::EdwardsPoint, traits::Identity};
+use ed25519_dalek::Verifier;
 use rand::thread_rng;
 
+fn verify_signature(
+    msg: &[u8],
+    group_signature: frost_core::Signature<Ed25519Sha512>,
+    group_pubkey: frost_core::VerifyingKey<Ed25519Sha512>,
+) {
+    let sig = {
+        let bytes: [u8; 64] = group_signature.to_bytes();
+        ed25519_dalek::Signature::from(bytes)
+    };
+    let pub_key = {
+        let bytes = group_pubkey.to_bytes();
+        ed25519_dalek::PublicKey::from_bytes(&bytes).unwrap()
+    };
+    // Check that signature validation has the expected result.
+    assert!(pub_key.verify(msg, &sig).is_ok());
+}
+
 #[test]
 fn check_sign_with_dealer() {
     let rng = thread_rng();
 
-    frost_core::tests::check_sign_with_dealer::<Ed25519Sha512, _>(rng);
+    // For the interoperability test it's better to test with multiple signatures
```

Comment:
Why is it better to test multiple signatures with the dealer but not with the DKG?

Reply:
My reasoning was that key generation does not really impact this, only the signature generation itself. It's better to generate multiple signatures and verify them, but how the key was generated is not that important. I chose the trusted dealer test because it's faster.

Thinking about this though, I think it wouldn't hurt to test the DKG itself multiple times, so I added a loop in 37f867f

Reply:
Makes sense, thanks for the clarification. I thought there was a deeper reason.